### DISCUSSION TITLE
Unwanted new blank tab

### DISCUSSION THREAD

**Discussion by prakhillohiya** [vercel/community#6450] [Help]
### Summary

Whenever a button is clicked anywhere on the UI, a new blank tab opens up which is redundant.

This is happening on Vercel as it works fine locally.

### Example

https://the-cv-builder.vercel.app/

### Steps to Reproduce

Click on the logo or Hi button
A new blank tab will be opened
This happens on all the buttons

**Comment by amyegan** [comment]
Hey @prakhillohiya. Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that we can use to help you debug this?

**Reply by prakhillohiya** [reply]
Hi,

You can reproduce the issue on the given deployment link above.

**Reply by amyegan** [reply]
It seems to be a code issue. I cannot see everything that is happening based on that link alone. If you would like help debugging the issue, we need to be able to work together from the same sample code.

**Reply by prakhillohiya** [reply]
Sure, below is the link of the repo

https://github.com/prakhillohiya/cv-builder/tree/main/frontend

**Reply by prakhillohiya** [reply]
Hi @amyegan, did you find something?

**Reply by prakhillohiya** [reply]
Hi @amyegan, any luck?

**Reply by prakhillohiya** [reply]
Hey @amyegan, It's been a long time, Please resolve this.
